docs/ibm_vpc.md

```diff
@@ -29,6 +29,7 @@ The IBM VPC client is a component for PyWren's docker executor using a remote ho
        compute_backend: docker
        storage_backend: ibm_cos
        auto_dismantle: True
```

Comment:
@kpavel is this True by default, for example when  auto_dismantle is not present in the configuration?